{
    "cveId": "CVE-2020-15801",
    "version": "1.0.0",
    "timestamp": "2024-12-13T17:34:55.850537+00:00",
    "description": "In Python 3.8.4, sys.path restrictions specified in a python38._pth file are ignored, allowing code to be loaded from arbitrary locations. The ._pth file (e.g., the python._pth file) is not affected.",
    "keyphrases": {
        "rootcause": "",
        "weakness": "",
        "impact": "",
        "vector": "load code from arbitrary locations",
        "attacker": "",
        "product": "Python",
        "version": "3.8.4",
        "component": ""
    }
}
